numpy/random/mtrand/mtrand.pyx

```diff
@@ -551,6 +552,19 @@ cdef object discd_array(rk_state *state, rk_discd func, object size, ndarray oa,
                 PyArray_MultiIter_NEXTi(multi, 1)
     return array
 
+cdef inline void bivariate_normal(double *rv_r, double *rv_i, double loc_r,
```

Comment:
Why inline?

Reply:
I used inline here since I saw some timing evidence that it mattered on Windows.